src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -3405,11 +3405,19 @@ static void Main()
         F((string s) => { });
     }
 }";
-            var comp = CreateCompilation(source);
-            comp.VerifyDiagnostics(
+
+            var expectedDiagnostics = new[]
```

Comment:
>var expectedDiagnostics = new[] [](http://example.com/codeflow?start=12&length=31)

There are no changes to the behavior of `OverloadResolution_17`, `_18`, `_19`, `_20`, but the tests have been expanded to test `-langversion:9` and `-langversion:10` explicitly.